src/app/builder/dragSection/drag-item/drag-item.component.scss

```
@@ -0,0 +1,50 @@
+$color-white: #ffffff;
```

create variables.scss and reuse
```suggestion
@import 'assets/scss/variables;
```